src/components/Dropdown/DropdownContentInner.tsx

```diff
@@ -61,15 +71,27 @@ export const DropdownContentInner: FC<Props> = ({
       contentBox={contentBox}
       scrollable={scrollable}
       className={`${className} ${isActive ? 'active' : ''}`}
+      controllable={controllable}
       themes={theme}
     >
-      {children}
+      {controllable ? (
+        children
+      ) : (
+        <DropdownContentInnerContext.Provider value={{ maxHeight: contentBox.maxHeight }}>
+          <DropdownCloser>{children}</DropdownCloser>
```

Comment:
[IMO] I thought that passing maxHeight as props would not increase the context.

Reply:
Yes, that's right. But doesn't it expose the maxHeight props? Then the user can pass any maxHeight value to DropdownCloser.

Reply:
If you don't want the user to touch maxHeight, it certainly looks good 👍

Reply:
It is not expected that the user will pass arbitrary values. I'll improve the components to accept arbitrary values if requested by the user 😄